bindings/web5_uniffi/src/web5.udl

```diff
@@ -257,10 +257,23 @@ dictionary VerifiableCredentialCreateOptionsData {
   sequence<string>? type;
   timestamp? issuance_date;
   timestamp? expiration_date;
+  CredentialSchemaData? credential_schema;
   string? json_serialized_evidence;
 };
 
 interface VerifiableCredential {
+  [Throws=Web5Error]
+  constructor(
```

Comment:
what's this second constructor for?

Reply:
good callout, this is unnecessary actually. thanks!